cmake/AddFdbTest.cmake

```diff
@@ -39,6 +39,9 @@ function(configure_testing)
 endfunction()
 
 function(verify_testing)
+  if(NOT ENABLE_SIMULATION_TESTS)
+    return()
+  endif()
```

Comment:
This whole block was a bug (see #6333). I don't think #6333 needs to be fully cherry-picked, but this code-block should be removed.

Reply:
Done